keras_cv/models/resnet_v2.py

```diff
@@ -389,6 +389,9 @@ def as_backbone(self):
 
         The output of the backbone model will be a dict with int as key and tensor as
```

Comment:
Are you sure it's a dict ? I see a list when I test ([repro colab](https://colab.research.google.com/drive/1-Fvf4sd0X0R14WNeC5899hho9B6hVHEG#scrollTo=Ezw0mHdT3tmy))

Reply:
It is a dict, the model.outputs flatten any structure into a dict. If you check model.output, then you will see the dict. You can also check the raw output if you give it a input tensor.